docs/spin.toml

```diff
@@ -1,198 +1,250 @@
+route = "/architecture"
+
+[[trigger.http]]
+id = "trigger-redirect-quickstart"
+component = "redirect-quickstart"
+route = "/quickstart"
+
+[[trigger.http]]
+id = "trigger-redirect-configuration"
+component = "redirect-configuration"
+route = "/configuration"
+
+[[trigger.http]]
+id = "trigger-redirect-developing"
+component = "redirect-developing"
+route = "/developing"
+
+[[trigger.http]]
+id = "trigger-redirect-rust-components"
+component = "redirect-rust-components"
+route = "/rust-components"
+
+[[trigger.http]]
+id = "trigger-redirect-go-components"
+component = "redirect-go-components"
+route = "/go-components"
+
+[[trigger.http]]
+id = "trigger-redirect-other-languages"
+component = "redirect-other-languages"
+route = "/other-languages"
+
+[[trigger.http]]
+id = "trigger-redirect-url-shortener"
+component = "redirect-url-shortener"
+route = "/url-shortener"
+
+[[trigger.http]]
+id = "trigger-redirect-http-trigger"
+component = "redirect-http-trigger"
+route = "/http-trigger"
+
+[[trigger.http]]
+id = "trigger-redirect-redis-trigger"
+component = "redirect-redis-trigger"
+route = "/redis-trigger"
+
+[[trigger.http]]
+id = "trigger-redirect-distributing-apps"
+component = "redirect-distributing-apps"
+route = "/distributing-apps"
+
+[[trigger.http]]
+id = "trigger-redirect-extending-and-embedding"
+component = "redirect-extending-and-embedding"
+route = "/extending-and-embedding"
+
+[[trigger.http]]
+id = "trigger-redirect-deploying-to-fermyon"
+component = "redirect-deploying-to-fermyon"
+route = "/deploying-to-fermyon"
+
+[[trigger.http]]
+id = "trigger-redirect-template-authoring"
+component = "redirect-template-authoring"
+route = "/template-authoring"
+
+[[trigger.http]]
+id = "trigger-redirect-sips-index"
+component = "redirect-sips-index"
+route = "/sips/index"
+
+[[trigger.http]]
+id = "trigger-redirect-sip001"
+component = "redirect-sip001"
+route = "/sips/001-spin-deploy.md"
+
+[[trigger.http]]
+id = "trigger-redirect-sip002"
+component = "redirect-sip002"
+route = "/sips/002-app-config.md"
+
+[[trigger.http]]
+id = "trigger-redirect-sip003"
+component = "redirect-sip003"
+route = "/sips/003-trigger-executors.md"
 
-[[component]]
-id = "redirect-download"
+[[trigger.http]]
+id = "trigger-redirect-sip004"
+component = "redirect-sip004"
+route = "/sips/004-spin-build.md"
+
+[[trigger.http]]
+id = "trigger-redirect-sip005"
+component = "redirect-sip005"
+route = "/sips/005-manifest-redesign.md"
+
+[[trigger.http]]
+id = "trigger-redirect-sip006"
+component = "redirect-sip006"
+route = "/sips/006-spin-plugins.md"
+
+[[trigger.http]]
+id = "trigger-redirect-contributing"
+component = "redirect-contributing"
+route = "/contributing"
+
+[component.redirect-download]
 source = "modules/redirect.wasm"
 environment = { DESTINATION = "https://developer.fermyon.com/downloads/install.sh" }
-[component.trigger]
-route = "/downloads/install.sh"
-executor = { type = "wagi" }
+allowed_outbound_hosts = ["https://developer.fermyon.com"]
```

Comment:
Are these needed?  They don't seem to be there in the v1 components.  I would expect us just to issue a 3xx to the client, not to need to actually do a request.

Reply:
Oh you are right - thank you.  Not needed; removed.